resources/views/admin/rejestracja.blade.php

```diff
@@ -1,178 +1,186 @@
+                         class="form-control" onchange="checkPassword()">
+                  <p class="help-block">Wprowadzone hasła muszą być identyczne</p>
+                </div>
+              </div>
+              <div class="form-group">
+                <div class="controls">
+                  <button class="btn btn-success" id="send_button">Zarejestruj</button>
+                </div>
+              </div>
+            </div>
+          </fieldset>
+        </form>
+      </div>
+    </div>
+  </div>
 
 
-<script>
-	function yesnoCheck(that) {
-        if (that.value == "pacjent") {
-            document.getElementById("gdyPacjent").style.display = "block";
-            document.getElementById("gdyPacjent1").style.display = "block";
-            document.getElementById("gdyPacjent2").style.display = "none";
-            document.getElementById("gdyPacjent3").style.display = "none";
-            document.getElementById("gdyPacjent4").style.display = "none";
-            document.getElementById("gdyPacjent5").style.display = "block";
 
-        } else {
-            document.getElementById("gdyPacjent").style.display = "none";
-            document.getElementById("gdyPacjent1").style.display = "none";
-            document.getElementById("gdyPacjent2").style.display = "block";
-            document.getElementById("gdyPacjent3").style.display = "block";
-            document.getElementById("gdyPacjent4").style.display = "block";
-            document.getElementById("gdyPacjent5").style.display = "none";
-        }
+  <script>
+    function yesnoCheck(that) {
+      if (that.value == 'pacjent') {
+        document.getElementById('gdyPacjent').style.display = 'block';
+        document.getElementById('gdyPacjent1').style.display = 'block';
+        document.getElementById('gdyPacjent2').style.display = 'none';
+        document.getElementById('gdyPacjent3').style.display = 'none';
+        document.getElementById('gdyPacjent4').style.display = 'none';
+        document.getElementById('gdyPacjent5').style.display = 'block';
+
+      } else {
+        document.getElementById('gdyPacjent').style.display = 'none';
+        document.getElementById('gdyPacjent1').style.display = 'none';
+        document.getElementById('gdyPacjent2').style.display = 'block';
+        document.getElementById('gdyPacjent3').style.display = 'block';
+        document.getElementById('gdyPacjent4').style.display = 'block';
+        document.getElementById('gdyPacjent5').style.display = 'none';
+      }
+    }
+
+    function checkName(input) {
+      const name = input.value;
+      const regex = /^[a-zA-Z]+$/;
+      toggleInvalidInput(input, regex.test(name));
     }
 
-</script>
+    function checkPesel(input) {
+      const pesel = input.value;
+      const regex = /^[0-9]{11}$/.test(pesel);
+
+      const isBadDate = (parseInt(pesel.substring(4, 6)) > 31) || (parseInt(pesel.substring(2, 4)) > 12);
```

Comment:
Wiem że to duperela ale osoby urodzone po 1999 roku mają do miesiąca w peselu dodawane 20.

Reply:
Ty, no przecież! Zapomniałem o tym całkowicie 😄 W XXI wieku mają dodawane 20, a już w XXII wieku będzie dodawane 40. Słuszna uwaga @LeoProXXX, już poprawiam 👍